src/evaluate.cpp

```diff
@@ -865,6 +852,19 @@ namespace {
            + Eval::Tempo;
   }
 
+/// Eval::init() initializes some tables needed by evaluation. A formula reduces
+/// independent parameters and allows easier tuning.
+void init() {
+
+  for (int m = 0; m < 32; ++m)
+  {
+    MobilityBonus[ QUEEN-2][m] = make_score(52-120*(1-log10(m+2.0)),  48-270*(1-log10(m+5.0)));
```

Comment:
is there a particular reason to write '52-120*(1-log10(m+2.0))' ? Equivalent would be '-64+120*log10(m+2.0)'

Reply:
That's just the first form that came into my head when I was trying to replicate the curves.  Your version (although not exact) is certainly formed better. . I will try to duplicate that form. . .